portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,28 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// Show current date and time on every page
+function showDate() {
+    const date = new Date();
+    const dateContainer = document.getElementById("date");
+    dateContainer.style.color = "white";
```

Comment:
Unless we're going to change the color based on the current date, it would be best to keep styling in the CSS.